akka-distributed-data/src/main/scala/akka/cluster/ddata/FastMerge.scala

```diff
@@ -23,7 +23,7 @@ private[akka] trait FastMerge { self: ReplicatedData ⇒
 
   private var ancestor: FastMerge = null
 
-  /** INTERNAL API: should be called from "updating" methods */
+  /** INTERNAL API: should be called from "updating" methods, and `resetDelta` */
```

Comment:
Use the brand new `@InternalApi` annotation also, for possible future mima-less-annoyingness.